frontend/src/employee-frontend/components/unit/unit-reservations/StaffAttendanceTable.tsx

```diff
@@ -478,6 +478,16 @@ const AttendanceRow = React.memo(function AttendanceRow({
   const { i18n } = useTranslation()
   const today = LocalDate.todayInHelsinkiTz()
 
+  const attendanceTooltipText = (attendance: Attendance) => (
+    <div>
+      {attendance.departedAutomatically ? (
+        <div>{i18n.unit.staffAttendance.departedAutomatically}</div>
+      ) : (
+        <div>{i18n.unit.staffAttendance.departedManually}</div>
```

Comment:
Is departedManually ever shown? Below the tooltip is omitted when `!attendance.departedAutomatically`. You can just return undefined here.